packages/cli/internal/pkg/cli/workflow/workflow_tasks.go

```diff
@@ -2,7 +2,8 @@ package workflow
 
 import (
 	"fmt"
-	"io"
+  "io"
```

Comment:
this might cause the CI to fail on formatting. If it does then try `make format` from the root of the project

Reply:
good call out. thanks.